src/herder/test/HerderTests.cpp

```diff
@@ -3809,10 +3848,10 @@ TEST_CASE("soroban txs accepted by the network",
 
             // Ensure upgrades went through
             simulation->crankForAtLeast(std::chrono::seconds(20), false);
+            for (auto node : nodes)
             {
-                LedgerTxn ltx(nodes[0]->getLedgerTxnRoot());
-                REQUIRE(nodes[0]
-                            ->getLedgerManager()
+                LedgerTxn ltx(node->getLedgerTxnRoot());
```

Comment:
I don't mind this change, but previous check was already sufficient: upgrades go through consensus, so if one node updated its state, the rest are guaranteed to do the same.